tests/dummy/app/templates/docs/transitions.md

```diff
@@ -0,0 +1,49 @@
+# Transitions
+
+Transitions assign each sprite to a motion. A transition tells sprite categories how to animate from one state to another. For example, a transition can assign the `fade` motion to all `insertedSprites`. Each time a sprite gets added to a list, it would fade into the list as that sprite went from its initial to its final state. 
+
+Transitions are interruptible and they run every time the state of a sprite changes. Therefore, they must be implemented as generator functions. Also, this means that the state of a sprite can change during a transition.
```

Comment:
I don't really know what a generator function is. A code example here would help. Following the "Also" sentence, it would be helpful to give a written example of why this is useful, i.e. "Also, this means that the state of a sprite can change during a transition. For example, if a user clicks an "undo" button as a sprite is changing states, the animation can reverse and the sprite could move back to its previous position."